### PR TITLE
lvutil: use wipefs not dd to clear existing signatures (xapi-project#624)

### PR DESCRIPTION
**Pull request by ydirson**

### CODE REVIEW

**Review by benjamreis — APPROVED** [review]

**Review by stormi — APPROVED** [review]

File: drivers/XE_SR_ERRORCODES.xml [hunk 1]
```diff
@@ -881,6 +881,12 @@
             <value>460</value>
         </code>
 
+        <code>
+            <name>WipeFS</name>
```
Comment (Wescoeur) [inline drivers/XE_SR_ERRORCODES.xml]: `WipefsFailure` maybe like other errors?

File: drivers/XE_SR_ERRORCODES.xml [hunk 2]
```diff
@@ -881,6 +881,12 @@
...
+            <description>Failed to wipe pre-existing filesystem signature.</description>
```
Comment (Wescoeur) [inline drivers/XE_SR_ERRORCODES.xml:886]: A lot of descriptions don't have a period at the end. Maybe we need to add a contribution in the upstream repository to standardize this situation.

File: drivers/util.py [hunk 3]
```diff
@@ -637,6 +637,17 @@ def zeroOut(path, fromByte, bytes):
     return True
 
 
+def wipefs(blockdev):
+    "Wipe filesystem signatures from `blockdev`"
+    cmdlist = ["/usr/sbin/wipefs", "-a", blockdev]
+    SMlog("Running command: %s" % cmdlist)
+    (rc, stdout, stderr) = doexec(cmdlist)
```
Comment (Wescoeur) [inline drivers/util.py]: `pread2(cmd)`?

File: drivers/XE_SR_ERRORCODES.xml (hunk 1 above)
Comment (ydirson) [reply drivers/XE_SR_ERRORCODES.xml]: done

File: drivers/util.py (hunk 3 above)
Comment (ydirson) [reply drivers/util.py]: done

**Review by Wescoeur — APPROVED** [review]

**Review by stormi — APPROVED** [review]

**Pull request merged**